### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by abgupta**
Specify max size of DL/SG/O365 group that can be imported in to a team.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ This article describes some of the limits, specifications, and other requirement
 |Number of channels per team    | 200 (includes deleted channels)&sup3;         |
 |Number of Private channels per team    |30| (includes deleted channels)&sup3;
 |Number of members in a Private channel    |250|
+|Maximum size of distribution list, security group or Office 365 group that can be imported in to a team    |3,500|
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:42]:
```suggestion
|Maximum size of a distribution list, security group, or Office 365 group that can be imported into a team    |3,500|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**